lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb

```diff
@@ -0,0 +1,31 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class FormattingChecksFlutterAction < Action
+      def self.run(params)
+        puts __dir__
```

Comment:
Looks like you left a `puts` here. Maybe use `UI.message` or remove it if it was for debug purposes ?

Reply:
oops, good catch! 🕵️